api/src/utils/test/fixtures/setting.ts

```diff
@@ -20,6 +20,14 @@ export const settingFixtures: SettingCreateDto[] = [
     type: SettingType.text,
     weight: 1,
   },
+  {
+    group: 'contact',
+    label: 'greeting',
+    value: 'hello',
+    type: SettingType.text,
+    weight: 10,
+    translatable: true,
+  },
```

Comment:
Do we need this for a given unit test ?

If yes, why don't we use existing ones ?

Reply:
those are unnecessary i removed them